addons/binding/org.openhab.binding.harmonyhub/ESH-INF/thing/thing-types.xml

```diff
@@ -32,6 +32,11 @@
                 <description>Heart beat keep alive time in seconds.
                 </description>
             </parameter>
+            <parameter name="ip" type="text" required="false">
+                <label>IP of Harmony hub (optional)</label>
```

Comment:
Other bindings usually call this "Network Address".
Additionally, you should add
```
                <context>network_address</context>
```